src/test/components/ProfileSummaryView.test.js

```diff
@@ -41,6 +23,40 @@ it('renders ProfileSummaryView correctly', () => {
     </Provider>,
     { createNodeMock }
   );
+  expect(profileSummary.toJSON()).toMatchSnapshot();
```

Comment:
actually I don't think "toJSON" is necessary, jest comes with a "react renderer snapshot serializer" or something.